chart-infra/templates/pod-template.tpl

```diff
@@ -0,0 +1,36 @@
+{{/* Generate a template that can be used for both assigned and unassigned xperiments */}}
+{{- define "pipeline.pod-template" -}}
+    metadata:
+      name: "{{ .Release.Name }}-server"
+      labels:
+        type: 'pipeline'
+        sandboxId: "{{ .Values.sandboxId }}"
+    spec:
+      restartPolicy: Always
```

Comment:
This makes a pipeline pod restart automatically when it fails, right? Is that what we want to do? Since when it fails we are showing the user it failed and not expecting it to rerun.

Or does this restart policy cause the pipeline to just restart but not rerun so it just rejoins the pool of available pipelines?

Reply:
`Always` is the **only** accepted value for pods. It will always restart so the deleton needs to be handled outside (see the kube-cleanup operator)